lib/poseidon/sync_producer.rb

```diff
@@ -57,6 +57,7 @@ def send_messages(messages)
         if !messages_to_send.pending_messages? || @max_send_retries == 0
           break
         else
+          Poseidon.logger.debug { "retrying sending #{messages_to_send.messages} messages"  }
```

Comment:
It's used for an exception message elsewhere, so I don't think it'll be serialized junk unless the value we're sending in are serialized junk. And if they are, we'll still get useful topic/partition info.